views/features.erb

```diff
@@ -73,6 +77,7 @@
 			<ul class="list list-bullet">
 				<li>You can check the status of the platform on our <a href="https://status.cloud.service.gov.uk/">system status page</a></li>
 				<li>Application logs can be streamed to SaaS logging platforms if service teams need to monitor app-specific performance metrics</li>
+				<li>Service metrics can be captured from backing services with our <a href="https://github.com/alphagov/paas-prometheus-exporter">Prometheus Exporter</a></li>
```

Comment:
I don't think a feature page needs an implementation-level detail like the specific tool used to talk a specific monitoring platform's protocol. YMMV.

Reply:
Removed reference to prometheus exporter